docs/linux/dm.md

```diff
@@ -431,7 +440,7 @@ Devices
   must be specified. For a multi-device target, a
   a list of block devices must be specified, with
   a regular block device as the first device specified,
-  followed by one or more zoned block devices
+  followed by one or more zoned block devices.
```

Comment:
ah! I need to patch dm-zoned-tools to add the period :)

Reply:
I'm glad to help find needles in haystacks.